homeassistant/components/honeywell/climate.py

```diff
@@ -25,9 +25,9 @@
 CONF_COOL_AWAY_TEMPERATURE = 'away_cool_temperature'
 CONF_HEAT_AWAY_TEMPERATURE = 'away_heat_temperature'
 
-DEFAULT_AWAY_TEMPERATURE = 16
-DEFAULT_COOL_AWAY_TEMPERATURE = 30
-DEFAULT_HEAT_AWAY_TEMPERATURE = 16
+DEFAULT_AWAY_TEMPERATURE = 16  # in C, for eu regions, the others are F/us
+DEFAULT_COOL_AWAY_TEMPERATURE = 88
```

Comment:
This feels very weird. Why would we take 1 temp in C, the other in F?

Reply:
It _is_ weird, I agree completely. That's why I made such an issue of it in https://github.com/home-assistant/home-assistant.io/pull/9587:
```
<p class='note'>
There is some potential confusion over this integration because...
```

(you'll recall I've just been asked to remove these warnings...)

The **honeywell** integration uses two distinct client libraries stuck together - one is evohome-client for EU/Celsius systems, (and uses `DEFAULT_AWAY_TEMPERATURE`), the other somecomfort for US/Fahrenheit systems (and uses the other two constants).  There was no reason for them to be the one integration - but they are.

Either system could be implemented as C or F, but one is almost always C, and the other is almost always F.

I am trying to sort it all out by deprecating the EU-half of **honeywell** in favour of **evohome** (the holdup is getting water_heater in - this has been a rather painful struggle for various reasons), and fixing anachronisms like this.

BTW, does this mean I can put this warning note back into the docs PR? ;)

Reply:
for example: https://github.com/home-assistant/home-assistant/issues/18932